frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -16,8 +16,11 @@ import { KeyboardArrowDown } from '@mui/icons-material'
 import { logOut } from '@fire'
 import { useAuthValue } from '@auth'
 import { useHistory } from 'react-router'
+import { DropdownSelect } from '@core/Components'
+import { SelectChangeEvent, Box } from '@mui/material'
 
 export const Dashboard = () => {
+  const [status, setStatus] = useState('')
```

Comment:
When I first read this I wasn't sure what `status` meant, maybe a variable name like `sortOrder` might be better?